src/snyk/service.ts

```diff
@@ -35,35 +35,62 @@ export class SnykService {
       throw new Error("Missing token for Snyk")
     }
 
-    const response = await fetch(
-      `https://snyk.io/api/v1/org/${encodeURIComponent(
-        snykAccountId,
-      )}/projects`,
-      {
+    let modifiedProjects: SnykProject[] = []
+
+    let nextUrl: string | undefined = `/orgs/${encodeURIComponent(
+      snykAccountId,
+    )}/projects?version=2023-08-04&meta.latest_dependency_total=true&meta.latest_issue_counts=true&limit=100`
```

Comment:
I recommend we have the version as a separate variable to make it easier to reference later on